style.css

```diff
@@ -0,0 +1,232 @@
+/* VARIABLES */
+:root {
+  --blue: #0077c5;
+}
+
+/* GLOBAL SETTINGS */
+body {
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  color: #505050;
+  letter-spacing: 0.2px;
+  background: #eceef1;
+}
+
+a {
+  text-decoration: none;
+  color: var(--blue);
+}
+
+p {
+  margin: 0;
+}
+
+button {
+  background-color: var(--blue);
+  border: none;
+  border: 1px solid #4c4c4c;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: 400;
+  cursor: pointer;
+  font-family: inherit;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+button .lock {
+  display: block;
+  width: 16px;
+  height: 14px;
+  background-image: url(images/lock.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 5px;
+}
+
+.blue {
+  color: var(--blue);
+  font-size: 12px;
+}
+
+/* TOP BAR */
+#top-bar {
+  display: flex;
+  justify-content: flex-end;
+  padding: 10px 30px;
+}
+
+#sign-in {
+  width: 290px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+#sign-in button {
+  width: 103px;
+  height: 34px;
+  font-size: 13.5px;
+}
+
+footer p {
+  margin: 5px auto;
+}
+
+#sign-in p {
+  margin: 0;
+  font-size: 12px;
+}
+
+#sign-in p:first-child {
+  margin-bottom: 2px;
+}
+
+/* GLOBAL WRAPPER */
+.wrapper {
+  text-align: center;
+  width: 100%;
+  margin: 0 auto;
+}
+
+/* TITLE */
+#logo {
+  width: 118px;
+  height: 35px;
+  padding-bottom: 20px;
+}
+
+#title {
+  padding: 30px 0 0 0;
+  text-align: center;
+}
+
+#icons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 23px;
+}
```

Comment:
[OPTIONAL] I understand you know this already but am just putting it here, refrain from using too many ID's. It's better to use classes in most cases as you could reuse them in the future. Its also because of specificity issues, for example: say you have an element with an ID that has a default color. To override this ID color with a class `.blue`, you'll have to increase `.blue`'s specificity and probably end up using `!important`, which is generally considered bad practice.